sorc/chgres_cube.fd/surface.F90

```diff
@@ -851,11 +851,12 @@ subroutine interp(localpet)
    if (localpet == 0) then   
      where(mask_target_one_tile == 1) mask_target_one_tile = 0
      where(mask_target_one_tile == 2) mask_target_one_tile = 1
+     call search_many(num_fields,bundle_seaice_target,tile, search_nums,localpet, &
```

Comment:
Compilation is failing here because field_data_3d is no longer an argument.

Reply:
Yes I'm working on debugging. I made changes on one system and wanted to get it to another to do some more work, so that's what that commit was. I've got more coming soon.